datadog/resource_datadog_timeboard.go

```diff
@@ -310,12 +318,16 @@ func buildTemplateVariables(terraformTemplateVariables *[]interface{}) *[]datado
 	return &datadogTemplateVariables
 }
 
-func appendRequests(datadogGraph *datadog.Graph, terraformRequests *[]interface{}) {
+func appendRequests(datadogGraph *datadog.Graph, terraformRequests *[]interface{}, service *interface{}) {
```

Comment:
I believe there's no need for a pointer to `interface{}` here since it's nullable itself.